src/script/arch-update.sh

```diff
@@ -190,8 +190,7 @@ invalid_option() {
 }
 
 # Definition of the icon directory
-icon_dir="/usr/share/icons/${name}"
-[ -d "/usr/local/share/icons/${name}" ] && icon_dir="/usr/local/share/icons/${name}"
+icon_dir="/usr/share/icons/hicolor/scalable/apps"
```

Comment:
Just for reference, while this is currently a breaking change for people installing Arch-Update with any other prefix than `/usr`, this will be dropped altogether in a following PR (we are dropping the changing icon in the `.desktop` file in favor of a future systray applet).